docs/user/ovirt/customization.md

```diff
@@ -0,0 +1,45 @@
+# Customizing a template during installation
+
+Before you begin, see the [generic cluster customization doc](https://github.com/openshift/installer/pull/docs/user/customization.md).
+
+You can customize cluster resources both during and after installing a cluster. This topic shows you how to customize a template during the installation process before the installer uses the template to create resources for the cluster.
+
+While creating a cluster, the OpenShift installer provides the following resources:
+
+- It uploads a qcow disk image of Red Hat Enterprise Linux CoreOS (RHCOS).
+- It creates a template from the pinned RHCOS qcow disk.
+- It creates master nodes and disks from the template.
+
+Customizing these installer-created resources before installation helps make the resource definitions consistent with each other. This approach is more straightforward than making customizations after installation. It also helps avoid issues caused by resources that are inconsistent with each other.
+
+The cluster also uses the template to create every master and worker node both during and after the installation.
+
+A dedicated VLAN network per cluster may come in the future.
+
+
+## Customizing install-config
+
+### Machine pools
+
+In progress - handled in https://bugzilla.redhat.com/1820300
+
+## Editing templates
+
+Optionally, you can customize the template the installer creates from the pinned RHCOS qcow disk.
+
+When the installer finishes creating the template, stop the installation for editing.
+Locate the template, named [infraID](https://github.com/openshift/installer/pull/pkg/asset/installconfig/clusterid.go)
+Edit the contents of the template. For example, you can edit it to create a node with two NICs.
+Save your edits to the template file.
```

Comment:
The env is designed for devs and not users of the product. Please either make it clear that there are plans to provide supported way of doing it, and this is a worksheet for time being.